docs/references/connections.md

```diff
@@ -0,0 +1,233 @@
+
+To create a connection, simply run [mp-test](mp-test.md) and then select `New Connection`. [mp-test](mp-test.md) will then create a new connection by
+parsing over the object model and using any [identifiers](../guides/adding_to_an_adapter.md#defining-an-adapter-and-adapter-instance-in-the-object-model)
+and [credentials](../guides/adding_to_an_adapter.md#defining-a-credential-in-the-object-model). The new connection is then stored in the
+[project config file](project_config.md) in the root of the project.
+
+
+#### Editing Existing Connections
+
+Connections can be edited by modifying the key-values in the [project config file](project_config.md) file at the root of the project that correspond to the value you want to modify.
+
+??? example
+
+    To edit the `container_memory_limit`  for connection with name `large-memory`, we can edit the key-value for
+    `container_memory_limit` inside the connection object (line 71).
+
+    ```json title="config.json" linenums="1" hl_lines="71"
+    {
+      "connections": [
+          {
+              "certificates": [],
+              "credential": {},
+              "custom_collection_number": null,
+              "custom_collection_window": null,
+              "identifiers": {
+                  "host": {
+                      "part_of_uniqueness": true,
+                      "required": true,
+                      "value": "example.com"
+                  },
+                  "container_memory_limit": {
+                      "part_of_uniqueness": false,
+                      "required": true,
+                      "value": "512"
+                  },
+                  "port": {
+                      "part_of_uniqueness": true,
+                      "required": true,
+                      "value": "443"
+                  }
+              },
+              "name": "small-memory",
+              "suite_api_hostname": null,
+              "suite_api_password": null,
+              "suite_api_username": null
+          },
+          {
+              "certificates": [],
+              "credential": {},
+              "custom_collection_number": null,
+              "custom_collection_window": null,
+              "identifiers": {
+                  "api": {
+                      "part_of_uniqueness": true,
+                      "required": true,
+                      "value": "example.com"
+                  },
+                  "container_memory_limit": {
+                      "part_of_uniqueness": false,
+                      "required": true,
+                      "value": "1024"
+                  },
+                  "port": {
+                      "part_of_uniqueness": true,
+                      "required": true,
+                      "value": "443"
+                  }
+              },
+              "name": "medium-memory",
+              "suite_api_hostname": null,
+              "suite_api_password": null,
+              "suite_api_username": null
+          },
```

Comment:
I think we could make these examples a bit smaller by removing the middle one, and then deleting either the small or large in the next example, without losing any context